packages/framework-provider-azure-infrastructure/src/infrastructure/helper/function-zip.ts

```diff
@@ -66,7 +70,20 @@ export class FunctionZip {
   static buildAzureFunctions(config: BoosterConfig): Array<FunctionDefinition> {
     const graphqlFunctionDefinition = new GraphqlFunction(config).getFunctionDefinition()
     const eventHandlerFunctionDefinition = new EventHandlerFunction(config).getFunctionDefinition()
-    let featuresDefinitions = [graphqlFunctionDefinition, eventHandlerFunctionDefinition]
+    const connectFunctionDefinition = new WebsocketConnectFunction(config).getFunctionDefinition()
+    const disconnectFunctionDefinition = new WebsocketDisconnectFunction(config).getFunctionDefinition()
+    const messagesFunctionDefinition = new WebsocketMessagesFunction(config).getFunctionDefinition()
+    let featuresDefinitions = [
+      graphqlFunctionDefinition,
+      eventHandlerFunctionDefinition,
+      connectFunctionDefinition,
+      disconnectFunctionDefinition,
+      messagesFunctionDefinition,
+    ]
+    const subscriptionsNotifierFunctionDefinition = new SubscriptionsNotifierFunction(config).getFunctionDefinition()
+    if (subscriptionsNotifierFunctionDefinition) {
+      featuresDefinitions = featuresDefinitions.concat(subscriptionsNotifierFunctionDefinition)
+    }
```

Comment:
Why only this function definition has the `if` check? I took a look at the method that generates it and it always return an array (an empty one if there are no read models), so, in theory, it should not be neccesary